content/messaging/agent-chat-api/v3.3/index.mdx

```diff
@@ -2197,40 +2197,55 @@ It returns the list of Customers.
-| `filters.agent_last_event_created_at.<date_range_filter_type>` **\*\*\*\***    | `object`  |                           |
-| `filters.customer_last_event_created_at.<date_range_filter_type>` **\*\*\*\*** | `object`  |                           |
+| Parameter                                                                        | Data type | Notes                     |
+|----------------------------------------------------------------------------------|-----------|---------------------------|
+| `page_id`                                                                        | `string`  |                           |
+| `limit`                                                                          | `number`  | Default: 10, maximum: 100 |
+| `sort_order` **\***                                                              | `string`  | Default: `desc`           |
+| `sort_by` **\*\***                                                               | `string`  | Default: `created_at`     |
+| `filters`                                                                        | `object`  |                           |
+| `filters.country.<string_filter_type>` **\*\*\***                                | `object`  |                           |
+| `filters.email.<string_filter_type>` **\*\*\***                                  | `object`  |                           |
+| `filters.name.<string_filter_type>` **\*\*\***                                   | `object`  |                           |
+| `filters.customer_id.<string_filter_type>` **\*\*\***                            | `object`  |                           |
+| `filters.chats_count.<range_filter_type>` **\*\*\*\***                           | `object`  |                           |
+| `filters.threads_count.<range_filter_type>` **\*\*\*\***                         | `object`  |                           |
+| `filters.visits_count.<range_filter_type>` **\*\*\*\***                          | `object`  |                           |
+| `filters.created_at.<date_range_filter_type>`**\*\*\*\*\***                      | `object`  |                           |
+| `filters.agent_last_event_created_at.<date_range_filter_type>` **\*\*\*\*\***    | `object`  |                           |
+| `filters.customer_last_event_created_at.<date_range_filter_type>` **\*\*\*\*\*** | `object`  |                           |
 
 **\*)**
 
 `sort_order` can take the following values:
 
-- `asc` - oldest customers first
-- `desc` - newest customers first
+- `asc`
+- `desc`
 
 **\*\*)**
 
+`sort_by` can take the following values:
+
+- `created_at`
+- `name`
+- `email`
+- `threads_count`
+- `visits_count`
+- `agent_last_event`
+- `customer_last_event`
+
+When sorting by `name`, `email`, `threads_count`, `visits_count`, `agent_last_event`, `customer_last_event` then `created_at` is used as secondary sorting factor.
```

Comment:
```suggestion
When sorting by fields other than `created_at`, the entries with identical values will be additionally sorted by their creation time.
```

Reply:
If we avoid listing all options by their name, we won't have to worry about updating this note when the options change their names, for example. 

"Secondary sorting factor" sounds a bit complicated.